tests/dtypes/test_geometry.py

```diff
@@ -135,3 +145,153 @@ def test_geometry_params_srid(
             'coordinates': 'POINT (582710 6061887)',
         }
     ]
+
+
+def test_geometry_html(rc: RawConfig):
+    context, manifest = load_manifest_and_context(rc, '''
+    d | r | b | m | property                   | type           | ref
+    example                                    |                |
+      |   |   | City                           |                |
+      |   |   |   | name                       | string         |
+      |   |   |   | coordinates                | geometry(4326) |
+    ''')
+    result = render_data(
+        context, manifest,
+        'example/City',
+        None,
+        accept='text/html',
+        data={
+            '_id': '19e4f199-93c5-40e5-b04e-a575e81ac373',
+            '_revision': 'b6197bb7-3592-4cdb-a61c-5a618f44950c',
+            'name': 'Vilnius',
+            'coordinates': 'POLYGON',
+        },
+    )
+    assert result == {
+        '_id': Cell(
+            value='19e4f199',
+            link='/example/City/19e4f199-93c5-40e5-b04e-a575e81ac373',
+            color=None,
+        ),
+        'name': Cell(
+            value='Vilnius',
+        ),
+        'coordinates': Cell(
+            value='POLYGON',
+            link=None,
+            color=None,
+        ),
+    }
+
+
+@pytest.mark.parametrize('value, cell', [
+    (
+        {
+            'geom_type': 'Point',
+            'geom_value': [582710, 6061887],
```

Comment:
These numbers do not look like WGS, maybe use:

```suggestion
            'geom_value': [25.28277, 54.68661],
```

?

Reply:
Values changed to similar in WGS values.